apache/incubator-zeppelin
# ZEPPELIN-527 Fix NPE while retrieving job status from notebook

### What is this PR for?

Fix NPE while retrieving job status from notebook which contains "never run" / "first run but not finished".

### What type of PR is it?
Bug Fix

### Todos

### Is there a relevant Jira issue?

https://issues.apache.org/jira/browse/ZEPPELIN-527

### How should this be tested?

1. Create a new notebook (```but don't run notebook!```)
2. Open ```http://<zeppelin host>:<zeppelin port>/api/notebook/job/<new notebook id>```
3. You can see HTTP STATUS 500 before this PR, but you can see HTTP STATUS 200 after this PR.

### Screenshots (if appropriate)

![2015-12-22 1 25 36](https://cloud.githubusercontent.com/assets/1317309/11947812/87605786-a8af-11e5-9d3f-3fda7efd12da.png)

### Questions:
* Does the licenses files need update? (No)
* Is there breaking changes for older versions? (No)
* Does this needs documentation? (No)
